### DISCUSSION TITLE
Nix geht mehr

### DISCUSSION THREAD

**Discussion by andesse** [Foddy/node-red-contrib-huemagic#296] [General]
Tot

**Comment by ddanier** [comment]
Ich kann mich hier nur anschließen! Danke für deine Arbeit. Hab eben auch 5$ gespendet 👍🤩

**Comment by Foddy** [comment]
Super! Ich danke Euch beiden :)!

**Comment by andesse** [comment]


**Reply by Foddy** [reply]
Kannst Du mir einen Hinweis geben, was sonst (außer den Events bei den Gruppen) nicht mehr bei dir funktioniert?

**Comment by s-n-k-r** [comment]
Zum Abschied noch einmal vielen Dank! Huemagic iVm. Redmatic (was einen ähnlichen Tod wie dieses Projekt gestorben ist) haben meinen Einstieg in die Hausautomatisierung geebnet. Ich finde es schade, dass beide Projekte so stillschweigend beendet wurden. Ich hatte Glück und bei mir lief bis zum Schluss alles mehr oder weniger stabil - trotz fehlender Anpassungen und Updates. Nichtsdestotrotz bin ich froh, dass ich jetzt nach der Migration auf Home Assistant wieder auf einer stabilen Plattform unterwegs bin, die regelmäßige Updates und Erweiterungen erhält.

**Reply by andesse** [reply]
Du hättest zu meinem Ansatz migrieren können. Gruß 

https://github.com/andesse/HUE-CLIP-API.Node-RED-Flows

**Reply by s-n-k-r** [reply]
Tatsächlich habe ich nicht nur den Code migriert, sondern die komplette Hardware und den gesamten DevOps Prozess. Vorher hatte ich 2 Hue Bridges (1 Bridge war mit 100 Geräten irgendwann überlastet) sowie 1 Tradfri Hub und 1 Aqara Hub im Einsatz. Dagegen scheint diese Anzahl von Geräten dem Home Assistant Yellow überhaupt nichts auszumachen. Und noch dazu laufen jetzt auch alle anderen Zigbee Geräte in einem einzigen großen Netzwerk und die Verzögerungen beim Schalten von Lampen sind auch weg. Und dank RedMatic-Addon und USB-Stick spare ich mir auch noch die Raspberrymatic als separate CCU.
Und das allerbeste, ich kann die Automatisierungen einfach in YAML schreiben, testen und per Gitlab mit HA syncen. Von daher war das alles ein guter Grund, die Hausautomatisierung insgesamt stabiler zu machen.
